lib/data_magic/config.rb

```diff
@@ -177,6 +181,21 @@ def column_field_types
       @column_types
     end
 
+    # actually does what column_field_types is meant to do - creates a hash of CSV column
```

Comment:
column_field_types does what it is meant to do... it's the field name for the column mapped to the type.... just in retrospect, it is easier to reason about the code if we just use column names OR full field names and omit the weird intermediary format.